Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -1,947 +0,0 @@
-/-
```

Comment:
Please use `deprecated_module` for this file. (Search mathlib for examples.) Thanks!